install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md

```diff
@@ -0,0 +1,155 @@
+# Migration Guide from Thanos to Kubecost 2.0 (Aggregator)
+
+This tutorial is intended to help our users migrate from the legacy Thanos federation architecture to [Kubecost 2.0 (Aggregator)](aggregator.md). There are a few requirements in order to successfully migrate to Kubecost 2.0. This new version of Kubecost includes a new backend Aggregator which handles the ETL data built from source metrics more efficiently. Kubecost 2.0 will provide new features, optimize the Kubecost's UI performance, and enhance the user experience.
+
+## Key changes
+
+* Assets and Allocations are now paginated
+   * Frontend/API users never have full set of information
+   * Use `offset`/`limit` parameters to govern pagination
+* New data available for querying every 2 hours (can be adjusted)
+* Embedded DuckDB database serves queries
+   * Data no longer queried directly from bingen files
+   * Substantial query speed improvements even when pagination not in effect
+* Data ingested into independent Aggregator component
+* Idle (sharing), Cluster Management sharing, and Network are computed a priori
```

Comment:
Another technical detail. IMO this shouldn't matter to a user (its just a technical trick for performance) but if you've gotten questions about this its fine to include.